readgssi.py

```diff
@@ -544,6 +544,27 @@ def readgssi(infile, outfile=None, antfreq=None, frmt=None, plot=False, figsize=
             for ar in chans:
                 a=[]
                 a=img_arr[(ar)*r[0]['rh_nsamp']:(ar+1)*r[0]['rh_nsamp']-timezero]
+                    
+                #Average Background Removal
+                meantrace=[0]*len(a[0])
+                for i in range(len(a)):
+                    for j in range(len(a[0])):
+                        meantrace[i]=a[i][j]+meantrace[i]
+                meantrace=[i/len(a[0])for i in meantrace]
+                
+                for j in range(len(a[0])):
+                    for i in range(len(a)):
+                        a[i][j]= a[i][j] - meantrace[i]                
+                
+                
+                #Dewow filter     
+                signal=zip(*a)[10]     
```

Comment:
The only change this needs for python3 compatibility is that instead of `signal=zip(*a)[10]` this should be `signal=list(zip(*a))[10]`